docroot/profiles/custom/sitenow/sitenow.profile

```diff
@@ -975,3 +976,25 @@ function sitenow_entity_insert(EntityInterface $entity) {
     }
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ *
+ * @param \Drupal\node\Entity\Node $node
+ *   Get straight to the node instead of through entity.
+ */
+function sitenow_node_presave(Node $node) {
+  $tags = ['node_type:' . $node->getType()];
+  Cache::invalidateTags($tags);
```

Comment:
Why are these necessary? Aren't these running for all node types, not just person?

Reply:
Figured we might use the node_type in the future for other content. the default node_list wasn't specific enough for me.

Reply:
I think the point here is that there should be a check by node type, so that it doesn't run for every type of node.

Reply:
Yea I don't know the reasoning behind this as it seems like default cache invalidation should work but I'm guessing this is going to result in A LOT more purges for all of our sites. I could test this theory if necessary but I think we might want to limit this to admissions person type specifically.

Reply:
I had to implement this because the map was not aware of node add/update/delete for person because it is getting person data differently than through views. The block calls this cache tag. Nothing else so far should be using it so I don't see a lot of purges based on this if it isn't misused.

Reply:
Oh gotcha. So this is a custom tag added in https://github.com/uiowa/uiowa/pull/2521/files#diff-7a4b444e952e83c3dd58f165a78332b15db945d70032fbaa8c2491bd7a912277R51?

I appreciate the abstraction but it still seems excessive to invalidate these tags for every node op on all sites when 90% of them don't use that tag. Considering how simple the hooks are to implement this for custom functionality, I would still vote limiting this to admissions.

Reply:
395eceb - moved to admissions and limit scope to person